pkg/skaffold/deploy/kpt.go

```diff
@@ -360,3 +348,82 @@ func getResources(root string) ([]string, error) {
 
 	return files, err
 }
+
+// getFlags returns a list of arguments that the user specified for the `kpt fn run` command.
+func (k *KptDeployer) getKptFnRunArgs() ([]string, error) {
+	// --dry-run sets the pipeline's output to STDOUT, otherwise output is set to sinkDir.
+	// For now, k.Dir will be treated as sinkDir (and sourceDir).
+	flags := []string{"--dry-run"}
+
+	if k.Fn.GlobalScope {
+		flags = append(flags, "--global-scope")
+	}
+
+	if len(k.Fn.Mount) > 0 {
+		flags = append(flags, "--mount", strings.Join(k.Fn.Mount, ","))
+	}
+
+	if k.Fn.Network {
+		flags = append(flags, "--network")
+	}
+
+	if len(k.Fn.NetworkName) > 0 {
+		flags = append(flags, "--network-name", k.Fn.NetworkName)
+	}
+
+	count := 0
+
+	if len(k.Fn.FnPath) > 0 {
+		flags = append(flags, "--fn-path", k.Fn.FnPath)
+		count++
+	}
+
+	if len(k.Fn.Image) > 0 {
+		flags = append(flags, "--image", k.Fn.Image)
+		count++
+	}
+
+	if count > 1 {
+		return nil, errors.New("only one of `fn-path` or `image` configs can be specified at most")
```

Comment:
sorry. can we change this to "only one of `fn-path` or `image` may be specified"

Reply:
of course!